src/test/java/software/amazon/cloudwatchlogs/emf/model/MetricDirectiveTest.java

```diff
@@ -90,6 +90,51 @@ void testPutMetricWithUnit() {
                 Unit.MILLISECONDS, metricDirective.getMetrics().get("Time").getUnit());
     }
 
+    @Test
+    void testPutMetricWithoutStorageResolution() throws JsonProcessingException {
+        MetricDirective metricDirective = new MetricDirective();
+        metricDirective.putMetric("Time", 10);
+
+        String serializedMetricDirective = objectMapper.writeValueAsString(metricDirective);
+
+        Assertions.assertEquals(
+                StorageResolution.STANDARD,
+                metricDirective.getMetrics().get("Time").getStorageResolution());
+        Assertions.assertEquals(
+                "{\"Dimensions\":[[]],\"Metrics\":[{\"Name\":\"Time\",\"Unit\":\"None\"}],\"Namespace\":\"aws-embedded-metrics\"}",
```

Comment:
Aren't we expecting the serialized emf contain, `"{\"Dimensions\":[[]],\"Metrics\":[{\"Name\":\"Time\",\"StorageResolution\":60,\"Unit\":\"None\"}],\"Namespace\":\"aws-embedded-metrics\"}",`

Irrespective of the putmetric call, I thought we have to pass StorageResolution:60 in the generated message.  Please correct me if my assumption is incorrect.

Reply:
No, actually in case of standard resolution, we will not serialize the StorageResolution parameter because 60 is the default value and adding that will result in unnecessary increase in payload and eventually increase latency.